README.md

```diff
@@ -1,49 +1,34 @@
-  * ConvNext-Base (W) @ 256x256 on LAION-A (900M sample aesthetic subset of 2B) with a top-1 of **71.0%**
-  * ConvNext-Base (W) @ 320x320 on LAION-A with a top-1 of **71.7%** (eval at 384x384 is **71.0**)
-  * ConvNext-Base (W) @ 320x320 /w augreg on LAION-A with a top-1 of **71.3%** (eval at 384x384 is **72.2%**)
-  * ConvNext-Large (D) @ 256x256 /w augreg on LAION-2B with a top-1 of **75.9%**
-  * ConvNext-Large (D) @ 320x320 fine-tune of 256x256 weights above for ~2.5B more samples on LAION-2B, top-1 of **76.6%**
-  * ConvNext-Large (D) @ 320x320 soup of 3 fine-tunes of 256x256 weights above on LAION-2B, top-1 of **76.9%**
-  * ConvNext-XXLarge @ 256x256 original run **79.1%** 
-  * ConvNext-XXLarge @ 256x256 rewind of last 10% **79.3%**
-  * ConvNext-XXLarge @ 256x256 soup of original + rewind **79.4%**
-
-Model cards w/ additional model specific details can be found on the Hugging Face Hub under the OpenCLIP library tag: https://huggingface.co/models?library=open_clip
-
-As we describe in more detail [below](#why-are-low-accuracy-clip-models-interesting), CLIP models in a medium accuracy regime already allow us to draw conclusions about the robustness of larger CLIP models since the models follow [reliable scaling laws](https://arxiv.org/abs/2107.04649).
-
-This codebase is work in progress, and we invite all to contribute in making it more accessible and useful. In the future, we plan to add support for TPU training and release larger models. We hope this codebase facilitates and promotes further research in contrastive image-text learning. Please submit an issue or send an email if you have any other requests or suggestions.
+Using this codebase, we have trained several models on a variety of data sources and compute budgets, ranging from [small-scale experiments](docs/LOW_ACC.md) to larger runs including models trained on datasets such as [LAION-400M](https://arxiv.org/abs/2111.02114), [LAION-2B](https://arxiv.org/abs/2210.08402) and [DataComp-1B](https://arxiv.org/abs/2304.14108).
+Many of our models and their scaling properties are studied in detail in the paper [reproducible scaling laws for contrastive language-image learning](https://arxiv.org/abs/2212.07143).
+Some of our best models and their zero-shot ImageNet-1k accuracy are shown below, along with the ViT-L model trained by OpenAI. 
+We provide more details about our full collection of pretrained models [here](docs/PRETRAINED.md), and zero-shot results for 38 datasets [here](docs/openclip_results.csv).
+
+
+| Model    | Training data | Resolution | # of samples seen | ImageNet zero-shot acc. | 
+| -------- | ------- |  ------- |  ------- |  ------- |  
+| ConvNext-Base | LAION-2B  | 256px | 13B | 71.5% |
+| ConvNext-Large | LAION-2B  | 320px | 29B | 76.9% |
+| ConvNext-XXLarge | LAION-2B | 256px | 34B | 79.5% |
+| ViT-B/32  | DataComp-1B  | 256px | 34B | 72.8% |
+| ViT-B/16  | DataComp-1B  | 224px | 13B | 73.5% |
+| ViT-L/14  | LAION-2B  | 224px | 13B | 75.3% |
```

Comment:
are you sure that's on 13B samples seen ? I thought that was on 32B. Did you check again the reproducibility paper ?

Reply:
yeah that l/14 is 32b

Reply:
indeed, updated